clients/java/signalr/src/main/java/HandshakeRequestMessage.java

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+public class HandshakeRequestMessage extends HubMessage{
```

Comment:
It does in the C# client.
https://github.com/aspnet/SignalR/blob/056da5114aeb201e805e8aa03eb9f6e32bdb9bf1/src/Microsoft.AspNetCore.SignalR.Common/Protocol/HandshakeResponseMessage.cs#L9

Reply:
Hrm, that's a bit sketchy since it isn't part of the Hub Protocol. I'm not sure why it does in C#, but I don't think we should continue that trend unless there's a very good reason to do it.

Reply:
also nit: spacing before `{`